index.html

```diff
@@ -1,2 +1,56 @@
 <!-- Saul Ernesto Castillo Chamagua-->
-
+<!DOCTYPE html>
+<html lang="en-us">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <title>Saul Castillo - Personal Portfolio</title>
+    </head>
+    <body>
+        <u>About:</u>    
```

Comment:
Header tags would be more semantically appropriate in place of the `<u>` tags. `<u>` tags are meant for annotating things like spelling errors https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u

Reply:
I agree, thanks for the advice Eric. Your help is greatly appreciated!